service_management/app/repositories/loan/dao/Tables.scala

```diff
@@ -0,0 +1,152 @@
+package repositories.loan.dao
+
+import java.sql.{Timestamp => JSqlTimestamp}
+
+import models.loan.LoanEventTypes.{ObjectLentType, ObjectReturnedType}
+import models.loan.LoanType
+import models.loan.event.{LoanEvent, ObjectsLent, ObjectsReturned}
+import no.uio.musit.models._
+import no.uio.musit.time.Implicits._
+import org.joda.time.DateTime
+import play.api.db.slick.HasDatabaseConfigProvider
+import play.api.libs.json.{JsValue, Json}
+import repositories.shared.dao.ColumnTypeMappers
+import slick.jdbc.JdbcProfile
+
+/**
+ * Defines the table definitions needed to work with analysis data.
+ * Also includes a few helpful functions to convert between rows and types.
+ */
+trait Tables extends HasDatabaseConfigProvider[JdbcProfile] with ColumnTypeMappers {
```

Comment:
Since we'll now get both loan and analysis in the same module, it's probably a good idea to given both the `Tables` traits specific names like `LoanTables` and `AnalysisTables`.